CONTRIBUTING.md

```diff
@@ -34,13 +34,14 @@ the team decides it is time for a new release, the process is to:
 
 1. Update `CHANGELOG.md` with the planned version number and a short bulleted
 list of major changes.  Include pull request numbers if applicable.
-1. Commit the changelog changes to master.
-1. Publish a new version with `apm publish <version>` where the version number
-follows semver conventions (`v<major>.<minor>.<patch>`).
+1. Run `npm run compile` to build the files in `lib/`
+1. Commit the changelog and `lib/` changes to master.
```

Comment:
"and" -> "and any" (If the earlier suggestion of running `npm run watch` was followed the changes should have already been built when the initial changes to `src` were made).